.github/labeler.yml

```diff
@@ -0,0 +1,66 @@
+# Add 'awaiting-review' to any PR opened
+awaiting-review:
+- '**/*'
+
+'product: CLion':
+- clwb/**/*
+- cpp/**/*
+- base/**/*
+- examples/cpp/simple_project/**/*
+- intellij_platform_sdk/BUILD.clion213
+- intellij_platform_sdk/BUILD.clion221
+- intellij_platform_sdk/BUILD.clion222
+- intellij_platform_sdk/BUILD.clion223
+- intellij_platform_sdk/BUILD.clion231
+- intellij_platform_sdk/BUILD.clion232
+
+'product: Android Studio':
+- aswb/**/*
+- sdkcompat/**/*
+- examples/kotlin/**/*
+- base/**/*
+- intellij_platform_sdk/BUILD.android_studio213
+- intellij_platform_sdk/BUILD.android_studio221
+- intellij_platform_sdk/BUILD.android_studio222
+- intellij_platform_sdk/BUILD.android_studio223
+- intellij_platform_sdk/BUILD.android_studio231
+- intellij_platform_sdk/BUILD.android_studiodev
+
+'product: IntelliJ':
+- aspect/**/*
+- base/**/*
+- build_defs/**/*
+- examples/java/greetings_project/**/*
+- ijwb/**/*
+- kotlin/**/*
+- plugin_dev/**/*
+- shared/**/*
+- java/**/*
+- third_party/java/jdk/**/*
```

Comment:
can we remove all `third_party` paths from this list?

Reply:
Same as above query?

Reply:
Sure